### DISCUSSION TITLE
Cannot power on Crazyflie when attached to flow deck

### DISCUSSION THREAD

**Discussion by yuhsiangsu** [bitcraze/discussions#656] [Q&A]
Hi,

When I attached the Flow Deck 2, I could not power on the Crazyflie, and only the M2 light was fully lit. I have already updated the firmware of the Crazyflie to the latest version. I am guessing it might be a problem with the firmware of the Flow Deck 2. However, I could not update the firmware for the Flow Deck 2 as I cannot power on the Crazyflie when attached. Is there any solution to this issue, please?

Thank you.

**Comment by tobbeanton** [comment]
Ahh a tricky problem. The Flow v2 does not have any firmware and my guess is that there is some form of HW problem. I guess you double checked that it is mounted in the right direction?

With a multimeter (if you have one) can you check the voltage of VCC when flow is connected and CF2.1 powered. That is GND on pin 1 and VCC on pin 10 on the left deck connector.
